pyfolio/plotting.py

```diff
@@ -30,18 +30,32 @@
 from . import pos
 
 
-def set_plot_defaults():
-    """
-    Sets default plotting/charting colors/styles.
-    """
+def plotting_context(func):
+    """Decorator to set plotting context during function call."""
+    def call_w_context(*args, **kwargs):
+        set_context = kwargs.pop('set_context', True)
+        if set_context:
+            with context():
+                return func(*args, **kwargs)
+        else:
+            return func(*args, **kwargs)
+    return call_w_context
+
+
+def context(context='notebook', font_scale=1.5, rc=None):
```

Comment:
Yeah, good point. I put that in for future use but there's currently no path to it. Your second idea is interesting but I'm worried that it breaks with the suggested way that these style context managers usually work. If I'm not mistaken, a user couldn't do `with set_style('my_style'): create_tear(set_style=False)` anymore but would have to pass the context in.